app/models/friendship.rb

```diff
@@ -0,0 +1,7 @@
+class Friendship < ApplicationRecord
+  validates :adder, presence: true
```

Comment:
We don't need to validate `adder` and `added`. The `belongs_to` associations are doing it for us.